### PR TITLE
move aurora modules

### LINKED ISSUE #24: move aurora modules

### PR DESCRIPTION
**Pull request by ryanking**
As part of this move, the modules have been refactored to have a common module. The -mysql and -postgres modules are still useful for setting appropriate defaults for each engine.

Fixes #24 #23

### CODE REVIEW

File: aws-aurora-postgres/main.tf [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+resource "aws_security_group" "rds" {
+  name        = "${local.name}"
+  description = "Allow postgres inbound"
+
+  vpc_id = "${var.vpc_id}"
+
+  ingress {
+    from_port   = "${var.database_port}"
```
Comment (edulop91) [inline aws-aurora-postgres/main.tf]: hmm maybe these shouldn't be vars. not currently propagated to the db resource.

**Comment by edulop91** [comment]
I think we should refactor into a common module - there's already some code drift between them that we probably don't want

File: aws-aurora-postgres/main.tf (hunk 1 above)
Comment (ryanking) [reply aws-aurora-postgres/main.tf]: That's weird, not sure how that would have worked before unless no one set them.

File: aws-aurora-mysql/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+output "database_name" {
+  value = "${module.aurora.database_name}"
+}
+
+output "endpoint" {
+  value = "${module.aurora.endpoint}"
+}
```
Comment (edulop91) [inline aws-aurora-mysql/outputs.tf]: maybe we should output the port too

File: aws-aurora-postgres/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,7 @@
+output "database_name" {
+  value = "${module.aurora.database_name}"
+}
+
+output "endpoint" {
+  value = "${module.aurora.endpoint}"
+}
```
Comment (edulop91) [inline aws-aurora-postgres/outputs.tf]: maybe we should output the port too

File: aws-aurora-mysql/outputs.tf (hunk 2 above)
Comment (ryanking) [reply aws-aurora-mysql/outputs.tf]: sure

**Pull request merged**